src/com/lftechnology/java/training/dipak/fibonacci/EvenFibonacci.java

```diff
@@ -0,0 +1,78 @@
+package com.lftechnology.java.training.dipak.fibonacci;
+
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+/*Each new term in the Fibonacci sequence is generated by adding the previous two terms. By starting with 1 and 2, the first 10 terms will be:
+
+1, 2, 3, 5, 8, 13, 21, 34, 55, 89, ...
+
+By considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms.
+*/

```

Comment:
if you use formatter your java doc will be breakdown. 
Format code with Ctrl+Shift+F and you will find what will happen
Use html <pre></pre> tag to format your code properly.
Author is missing in class level java doc
Do not add new line between java doc and method/class declaration.